DigitalOffice/Services/UserService/Validators/EditUserRequestValidator.cs

```diff
@@ -0,0 +1,40 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.UserService.Models;
+
+namespace LT.DigitalOffice.UserService.Validators
+{
+    public class EditUserRequestValidator : AbstractValidator<EditUserRequest>
+    {
+        public EditUserRequestValidator()
```

Comment:
Add validation rule for Id and add test for it

Reply:
Fixed